templates/js-start-scrape-single-page/src/main.js

```diff
@@ -0,0 +1,42 @@
+// Axios import - Promise based HTTP client for the browser and node.js (Read more at https://axios-http.com/docs/intro)
+import axios from "axios";
+
+// Cheerio - The fast, flexible & elegant library for parsing and manipulating HTML and XML (Read more at https://cheerio.js.org/)
+import * as cheerio from 'cheerio';
+
+// Apify SDK - toolkit for building Apify Actors (Read more at https://docs.apify.com/sdk/js/)
+import { Actor } from "apify";
+
+// Initialize the Apify SDK
+await Actor.init();
+
+// Structure of input is defined in input_schema.json.
+const input = await Actor.getInput();
+
+// Getting the URL from input defined in ./storage/key_value_stores/default/INPUT.json
+const { url } = input;
```

Comment:
I have two separate comments to this:

1. I don't think we should describe simple operations on variables.
2. When you create the actor in console, there won't be any storage folder, so the comment is actually misleading.